### PR TITLE
juan-section-11 updated section 11 and some comments in other sections

### PR DESCRIPTION
**Pull request by JuanDelahozMiranda**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -602,79 +607,86 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+        this.itemBackpack_AddBtn = "[data-test=\"add-to-cart-sauce-labs-backpack\"]";
+        this.titleItem = "#item_4_title_link > .inventory_item_name";
+        this.priceItem = ":nth-child(1) > .inventory_item_description > .pricebar > .inventory_item_price";
+    }
 
-     constructor(){
-       this.dressItem = ".product-container"
-       this.dressName = `${this.dressItem} .product-name`
-     }
+    public goToShoppingcar():void{
```
Comment (luisdanielvcord) [inline README.md]: There's a small typo on the function name, it should be "goToShoppingCart()" with a capital "C" and a "t" at the end

File: README.md [hunk 2]
```diff
@@ -602,79 +607,86 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+    private itemBackpack_AddBtn: string;
+    private titleItem: string;
+    private priceItem: string;
+
+    constructor() {
+        this.shoppingbtn = ".shopping_cart_link";
+        this.containeritems = ":nth-child(2) > :nth-child(1) > #inventory_container";
+        this.itemBackpack_AddBtn = "[data-test=\"add-to-cart-sauce-labs-backpack\"]";
+        this.titleItem = "#item_4_title_link > .inventory_item_name";
+        this.priceItem = ":nth-child(1) > .inventory_item_description > .pricebar > .inventory_item_price";
```
Comment (luisdanielvcord) [inline README.md]: Since we improved the selectors in section 10, Shouldn't we provide ideal selectors for this example?

File: README.md [hunk 3]
```diff
@@ -602,79 +607,86 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
-     }
+    public verifyTitle(messages:string): void {
+        cy.get(this.titleItem).should("have.text", messages);
+    }
 
-     validateItemsNames(names: string[]){
-       cy.get(this.dressName).each((item, index) => {
-         cy.wrap(item).should("contain.text", names[index])
-       })
-     }
```
Comment (luisdanielvcord) [inline README.md:640]: By removing the validateItemsNames method we are reducing code complexity, Is this expected?

File: README.md [hunk 4]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+11.1. Primero verifique que cuenta con las clases para cada page usando Page Object (recuerde que al crear un `nombre/pagina.page.ts` que contenga los selectores, debe ser agregado al `index.ts`).
 
-   ```javascript
-   class DressesListPage {
-
-     private dressItem: string;
-     private dressName: string;
+   ```js
+   class ProductsContentPage {
+    private shoppingbtn: string
```
Comment (diegobonilla86) [inline README.md]: Good practice to name variables:

private shoppingBtn: string

File: README.md [hunk 5]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+    private containeritems: string;
```
Comment (diegobonilla86) [inline README.md]: private containerItems: string;

File: README.md [hunk 6]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+    private itemBackpack_AddBtn: string;
+    private titleItem: string;
+    private priceItem: string;
+
+    constructor() {
+        this.shoppingbtn = ".shopping_cart_link";
```
Comment (diegobonilla86) [inline README.md]: this.shoppingBtn = ".shopping_cart_link";

File: README.md [hunk 7]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+        this.containeritems = ":nth-child(2) > :nth-child(1) > #inventory_container";
```
Comment (diegobonilla86) [inline README.md]: this.containerItems = ":nth-child(2) > :nth-child(1) > #inventory_container";

File: README.md [hunk 8]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+        this.itemBackpack_AddBtn = "[data-test=\"add-to-cart-sauce-labs-backpack\"]";
```
Comment (diegobonilla86) [inline README.md]: this.itemBackPack_AddBtn = "[data-test=\"add-to-cart-sauce-labs-backpack\"]";

File: README.md [hunk 9]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+        this.titleItem = "#item_4_title_link > .inventory_item_name";
+        this.priceItem = ":nth-child(1) > .inventory_item_description > .pricebar > .inventory_item_price";
+    }
 
-     constructor(){
-       this.dressItem = ".product-container"
-       this.dressName = `${this.dressItem} .product-name`
-     }
+    public goToShoppingcar():void{
```
Comment (diegobonilla86) [inline README.md]: public goToShoppingCart():void{

File: README.md [hunk 10]
```diff
@@ -602,79 +607,88 @@ En esta sección deberas personalizar los selectores que se estan usando para la
...
+        cy.get(this.shoppingbtn).click();
```
Comment (diegobonilla86) [inline README.md]: cy.get(this.shoppingBtn).click();

**Review by diegobonilla86 — CHANGES_REQUESTED** [review]
Please rename variables and methods according to the comments

File: README.md (hunk 3 above)
Comment (JuanDelahozMiranda) [reply README.md:640]: in this case the method was based in the before page, so is updated

File: README.md (hunk 1 above)
Comment (JuanDelahozMiranda) [reply README.md]: ok

**Review by luisdanielvcord — APPROVED** [review]

**Review by diegobonilla86 — APPROVED** [review]

File: README.md [hunk 11]
```diff
@@ -75,8 +75,8 @@ Ten en cuenta tener estudiados ciertos conceptos importantes (te dejamos unos en
 2.1. Instalar la versión `v16.X.0` (o superiores) de Node.js.
 
    **Notas:**
-   - Recomendamos usar [nvm](https://github.com/nvm-sh/nvm) como manejador de versiones.
-   - Necesariamente no tienes que instalar la version 16, pero si recomendamos instalar una version LTS, hemos probado este workshop con las versiones `v14.X.0` ó `v16.X.0`.
+   - Recomendamos usar [nvm](https://github.com/nvm-sh/nvm) como manejador de versiones (Opcional).
```
Comment (davidmoralesv) [inline README.md:78]: Esto confunde tanto que una persona que comienza el workshop, va a instalar nvm para luego darse cuenta que no lo va a necesitar. Además que está fuera de lugar porque todo el tiempo se usa npm.

**Pull request merged**